src/main/psxsdk/libsnd/vmanager.c

```diff
@@ -189,7 +193,21 @@ short SsUtChangePitch(short voice, short vabId, short prog, short old_note,
     return -1;
 }
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/vmanager", SsUtChangeADSR);
+s16 SsUtChangeADSR(
```

Comment:
Maybe it's a personal preference, but maybe using a guard clause makes it more readable? https://decomp.me/scratch/tCdUq

Also notice the use of `LEN(_svm_voice)` instead of the `24`

Reply:
I added NUM_SPU_CHANNELS since there's a bunch of different arrays of that length. I think the nested way is more like how my brain thinks but if others prefer the early returns we can do that.